modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/DhsClient.scala

```diff
@@ -204,7 +204,7 @@ class DhsClientSim(date: LocalDate) extends DhsClient {
 
   override def createImage(p: ImageParameters): SeqAction[ImageFileId] =
     EitherT(Task.delay{
-      TrySeq(f"S${date.format(format)}${counter.incrementAndGet()}%04d")
+      TrySeq(f"S${date.format(format)}S${counter.incrementAndGet()}%04d")
```

Comment:
I am not sure what the second is for (Summit/Base ?). But this is DhsClientSim, I don't think it needs more work.